src/app/components/app-layout/app-layout.component.scss

```diff
@@ -115,3 +127,38 @@
     width: 100%;
   }
 }
+
+.dropdown-menu {
+  left: auto!important;
+  right: 0px!important;
+
+  a {
+    cursor: pointer;
+  }
+}
+
+.btn-primary {
+  .show { }
+  .dropdown-toggle {
+    &, &:focus { }
+  }
+  // etc
```

Comment:
My suggestion was an example, has this been tested? If it works just like this, these empty statements can be removed.